mkosi/architecture.py

```diff
@@ -121,11 +121,16 @@ def default_serial_tty(self) -> str:
         return {
             Architecture.arm   : "ttyAMA0",
             Architecture.arm64 : "ttyAMA0",
+            Architecture.s390  : "ttysclp0",
```

Comment:
Do we need s390? While s390x is alive, I remember s390 being a dead arch like itanium or superh or m68k

Reply:
It is a part of the discoverable partitions spec, so I don't think it hurts to do minimal effort to try and support it. I would treat it the same as s390x in all cases.